wasm/lib/src/browser_module.rs

```diff
@@ -361,11 +263,10 @@ pub fn make_module(vm: &VirtualMachine) -> PyObjectRef {
 
     let element = Element::make_class(ctx);
 
-    py_module!(vm, "browser", {
+    py_module!(vm, "_browser", {
```

Comment:
Ok this allows now for an `import _browser` in WASM.

But what are the differences between the browser (frozen) and the browser (built-in) module, they don't seem to be the same:
```
>>>> dir(browser)
['Document', 'Element', 'JSValue', 'Promise', '__annotations__', '__builtins__', '__doc__', '__loader__', '__name__', '__package__', '__spec__', '_alert', '_confirm', '_prompt', 'alert', 'cancel_animation_frame', 'confirm', 'document', 'fetch', 'jsclosure', 'jsstr', 'load_module', 'prompt', 'request_animation_frame', 'window']
>>>>> dir(_browser)
['Document', 'Element', '__doc__', '__loader__', '__name__', '__package__', '__spec__', 'cancel_animation_frame', 'document', 'fetch', 'load_module', 'request_animation_frame']
>>>>> browser
<module 'browser' (frozen)>
>>>>> _browser
<module '_browser' (built-in)>
```

Reply:
It doesn't actually change anything but `import _browser; _browser.__name__`, if you look at the source of `browser.py` you can see that it imports `*` from `_browser`. `_browser` has stuff like fetch and document, `browser` wraps it and also provides some other functions that are easy to implement in Python. It's like the built-in `posix`/`nt` vs the Python `os`.

Reply:
[For reference](https://github.com/RustPython/RustPython/blob/master/wasm/lib/src/browser.py)